patitasuy/src/app/services/posts.service.ts

```diff
@@ -1,34 +1,16 @@
 import { Injectable } from '@angular/core';
 import { delay, map, Observable, of } from "rxjs";
-import { Post } from "../models/post.interface";
-import { HttpClient } from "@angular/common/http";
-import { environment } from "../../environments/environment";
+import { Post } from "../interfaces/post.interface";
+import {HttpClient} from "@angular/common/http";
+import {environment} from "../../environments/environment";
 
 @Injectable({
     providedIn: 'root'
 })
 export class PostsService {
 
     private readonly apiUrl = environment.apiUrl + '/posts';
-
-    fakePost: Post = {
-        id: 1,
-        title: 'Post de prueba',
-        url: [
-            'https://images.pexels.com/photos/3687770/pexels-photo-3687770.jpeg',
-            'https://images.pexels.com/photos/2607544/pexels-photo-2607544.jpeg',
-            'https://images.pexels.com/photos/3687770/pexels-photo-3687770.jpeg',
-            'https://images.pexels.com/photos/2607544/pexels-photo-2607544.jpeg',
-            'https://images.pexels.com/photos/3687770/pexels-photo-3687770.jpeg',
-        ],
-        description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquid autem cum eaque id incidunt iure libero, molestias nulla obcaecati odit provident quaerat quasi quidem reprehenderit repudiandae sequi temporibus velit voluptates. texto jaja hola buenas pedo culo caca pis',
-        age: 1,
-        sex: 'Macho',
-        type: 'dog',
-        size: 'Mediano',
-        location: 'Montevideo, Uruguay',
-        state: "Activo"
-    };
+    private readonly apiUrl2 = environment.apiUrl + '/post';
```

Comment:
This should not be necessary, just use "apiUrl"

Reply:
done, thanks